google-ads-migration-examples/src/main/java/com.google.ads.googleads.migration/createcampaign/CreateCompleteCampaignAdWordsApiOnly.java

```diff
@@ -0,0 +1,429 @@
+import com.google.api.ads.adwords.axis.v201809.cm.AdvertisingChannelType;
+import com.google.api.ads.adwords.axis.v201809.cm.BiddableAdGroupCriterion;
+import com.google.api.ads.adwords.axis.v201809.cm.BiddingStrategyConfiguration;
+import com.google.api.ads.adwords.axis.v201809.cm.BiddingStrategyType;
+import com.google.api.ads.adwords.axis.v201809.cm.Bids;
+import com.google.api.ads.adwords.axis.v201809.cm.Budget;
+import com.google.api.ads.adwords.axis.v201809.cm.BudgetBudgetDeliveryMethod;
+import com.google.api.ads.adwords.axis.v201809.cm.BudgetOperation;
+import com.google.api.ads.adwords.axis.v201809.cm.BudgetReturnValue;
+import com.google.api.ads.adwords.axis.v201809.cm.BudgetServiceInterface;
+import com.google.api.ads.adwords.axis.v201809.cm.Campaign;
+import com.google.api.ads.adwords.axis.v201809.cm.CampaignOperation;
+import com.google.api.ads.adwords.axis.v201809.cm.CampaignReturnValue;
+import com.google.api.ads.adwords.axis.v201809.cm.CampaignServiceInterface;
+import com.google.api.ads.adwords.axis.v201809.cm.CampaignStatus;
+import com.google.api.ads.adwords.axis.v201809.cm.CpcBid;
+import com.google.api.ads.adwords.axis.v201809.cm.ExpandedTextAd;
+import com.google.api.ads.adwords.axis.v201809.cm.Keyword;
+import com.google.api.ads.adwords.axis.v201809.cm.KeywordMatchType;
+import com.google.api.ads.adwords.axis.v201809.cm.Money;
+import com.google.api.ads.adwords.axis.v201809.cm.NetworkSetting;
+import com.google.api.ads.adwords.axis.v201809.cm.Operator;
+import com.google.api.ads.adwords.axis.v201809.cm.UrlList;
+import com.google.api.ads.adwords.axis.v201809.cm.UserStatus;
+import com.google.api.ads.adwords.lib.factory.AdWordsServicesInterface;
+import com.google.api.ads.common.lib.conf.ConfigurationLoadException;
+import com.google.api.ads.common.lib.exception.OAuthException;
+import com.google.api.ads.common.lib.exception.ValidationException;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.ads.adwords.lib.client.AdWordsSession;
+import com.google.api.ads.common.lib.auth.OfflineCredentials;
+import com.google.api.ads.common.lib.auth.OfflineCredentials.Api;
+import java.io.UnsupportedEncodingException;
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.net.URLEncoder;
+
+import org.joda.time.DateTime;
+
+import static com.google.api.ads.common.lib.utils.Builder.DEFAULT_CONFIGURATION_FILENAME;
+
+/**
+ * This code example is the first in a series of code examples that shows how to create a Search
+ * campaign using the AdWords API, and then migrate it to the Google Ads API one functionality at a
+ * time. See other examples for code examples in various stages of migration.
+ *
+ * This code example represents the initial state, where the AdWords API is used to create a
+ * campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
```

Comment:
Is the last sentence meant to indicate that in this scenario, the _user_ has not migrated the functionality to the Google Ads API? The way it's written, you could interpret it as the Google Ads API does not have that functionality.

Reply:
I have changed the sentence to clarify that the user has not migrated.